packages/vm/tests/api/istanbul/eip-2200.js

```diff
@@ -46,7 +46,7 @@ tape('Istanbul: EIP-2200: net-metering SSTORE', async t => {
     await vm.stateManager.putAccount(addr, account)
     await vm.stateManager.putContractCode(addr, Buffer.from(testCase.code, 'hex'))
     if (!testCase.original.isZero()) {
-      await vm.stateManager.putContractStorage(addr, key, testCase.original)
+      await vm.stateManager.putContractStorage(addr, key, testCase.original.toBuffer())
```

Comment:
@evertonfraga The browser tests on this PR are hanging, this might be the culprit?

Reply:
Seems to be